src/CMakeLists.txt

```diff
@@ -796,6 +800,10 @@ target_link_libraries(ceph-dencoder
   ${EXTRALIBS}
   ${CMAKE_DL_LIBS}
   )
+if(${WITH_KVS})
```

Comment:
@tchaikov 
Not sure how to divided squashed commits back into loose parts.
For me it is easier to start fresh I think.